ecosystem/sep-0024.md

```diff
@@ -6,8 +6,8 @@ Title: Hosted Deposit and Withdrawal
 Author: SDF
 Status: Active
 Created: 2019-09-18
-Updated: 2022-05-05
-Version 2.3.0
+Updated: 2022-05-06
+Version 2.3.1
```

Comment:
Since this is a new feature, the addition of a new status, this should be a minor version bump. Reserve patch version bumps for bug fixes.
```suggestion
Version 2.4.0
```

Reply:
Updated and pushed.